app/controllers/api/v1/merchants_controller.rb

```diff
@@ -1,16 +1,34 @@
 class Api::V1::MerchantsController < ApplicationController
   def index
+    render json: MerchantSerializer.new(Merchant.all)
   end
 
   def show
+    render json: MerchantSerializer.new(Merchant.find(params[:id]))
   end
 
   def create
+    render json: MerchantSerializer.new(Merchant.create(merchant_params))
   end
 
   def update
+    render json: MerchantSerializer.new(Merchant.update(params[:id], merchant_params))
   end
 
   def destroy
```

Comment:
Lots of dependents for the merchant. Using postman as a tool to test the status of this endpoint was SO helpful.